exporter/loadbalancingexporter/README.md

```diff
@@ -21,9 +21,17 @@ Note that either the Trace ID or Service name is used for the decision on which
 
 This load balancer is especially useful for backends configured with tail-based samplers or red-metrics-collectors, which make a decision based on the view of the full trace.
 
-When a list of backends is updated, around 1/n of the space will be changed, so that the same trace ID might be directed to a different backend, where n is the number of backends. This should be stable enough for most cases, and the higher the number of backends, the less disruption it should cause. Still, if routing stability is important for your use case and your list of backends are constantly changing, consider using the `groupbytrace` processor. This way, traces are dispatched atomically to this exporter, and the same decision about the backend is made for the trace as a whole.
+When a list of backends is updated, some of the signals will be rerouted to different backends. 
+Around R/N of the "routes" will be rerouted differently, where:
```

Comment:
I might be missing something, but it would be more like 1/R, according to Karger et al., which is the paper this algorithm is based on. The message is that not all trace IDs will land on different backends, only a portion of them.

Reply:
The Wikipedia page on [Consistent Hashing](https://en.wikipedia.org/wiki/Consistent_hashing) also mentions Karger, and describes the algorithm like this:

> The term "consistent hashing" was introduced by David Karger et al. at MIT for use in distributed caching, particularly for the web. This academic paper from 1997 in Symposium on Theory of Computing introduced the term "consistent hashing" as a way of distributing requests among a changing population of web servers. Each slot is then represented by a server in a distributed system or cluster. The addition of a server and the removal of a server (during scalability or outage) requires only `num_keys/num_slots` items to be re-shuffled when the number of slots (i.e. servers) change.

To me, `R/N` makes intuitive sense for these reasons:
* The more "routes" we have, the more routes will be reshuffled during an addition/removal of a backend.
* The more "backends" we have, the less routes we have to reshuffle during an addition/removal of a backend.

Reply:
It does make sense! I'm OK with this change as it is.